docs/config/README.md

```diff
@@ -2399,6 +2399,10 @@ The module will be shown if any of the following conditions are met:
 - The user is currently connected as an SSH session
 - The variable `show_always` is set to true
 
+Note: SSH connection is detected by checking environment variables
+`SSH_CONNECTION`, `SSH_CLIENT`, and `SSH_TTY`. If your SSH host does not set up
+these variables, one workaround is to set one of them with a dummy value.
+
```

Comment:
You can make this stand out a bit more by marking this as a tip (hopefully I've got the syntax right here). 

```suggestion
:::: tip
SSH connection is detected by checking environment variables
`SSH_CONNECTION`, `SSH_CLIENT`, and `SSH_TTY`. If your SSH host does not set up
these variables, one workaround is to set one of them with a dummy value.
::::

```